docs/guide/util/common.md

```diff
@@ -1,9 +1,27 @@
-# Coming Soon
+# Common Helpers
 
-::: info
-This page is Under Construction 🏗️, whilst Moonwall has been built - the docs have not.
+## Artifact Downloader
 
-Please Stay tuned for more details of this feature!
-:::
+Moonwall has an integrated artifact downloader that allows you to quickly grab client artifacts that you need, without having to leave the CLI. 
 
-![underconstruction](/under-construction.png)
+### Using the Artifact Downloader
+
+1. Choose option **4. Artifact Downloader** from the Moonwall main menu.
+2. Select from the desired network
+3. Select the desired version
+4. Enter the directory where you'd like the artifact placed 
+5. Review your selections and press enter to start the download
+
+When the download is completed, Moonwall will attempt to launch the binary, but your operating system may prevent this by default. You can mark the file as executable and run the binary for the respective network as you usually do, or Moonwall can run it for you when running your tests.
+
+![screenshot of artifact downloader](/artifact-downloader.png)
+
+### Supported Networks
+
+Artifact Downloader currently supports downloading artifacts for the following networks:
+
+- Moonbeam
+- Polkadot
+- Tanssi
+
+Would you like to see other networks added here? Let us know at info@moonsonglabs.com or file an issue in the Github repo. 
```

Comment:
We actually have support for adding new networks via the moonwall config file!

Check out some examples in the: [test folder](https://github.com/Moonsong-Labs/moonwall/blob/9431e874f33c164a65c6d3554d650c90eb8d4b86/test/moonwall.config.json#L6)

----


For a more permanent addition, networks can raise a PR to add a new networkSpec file: [here](https://github.com/Moonsong-Labs/moonwall/tree/main/packages/cli/src/lib/repoDefinitions)

here is `tanssi` which should hopefully be self-expanatory:
```js
mport { RepoSpec } from "@moonwall/types";

const repo: RepoSpec = {
  name: "tanssi",
  binaries: [
    { name: "tanssi-node", defaultArgs: ["--dev", "--sealing=manual", "--no-hardware-benchmarks"] },
    { name: "container-chain-template-simple-node" },
    { name: "container-chain-template-frontier-node" },
  ],
  ghAuthor: "moondance-labs",
  ghRepo: "tanssi",
};
export default repo;
```